src/main/java/com/mealmentor/enterprise/service/IMealPlanService.java

```diff
@@ -14,14 +14,39 @@
      */
     MealItem fetchById(int id);
 
+    /**
+     * Save a mealItem
+     * @param mealItem
+     * @return MealItem
+     * @throws Exception
+     */
     MealItem save(MealItem mealItem) throws Exception;
 
+    /**
+     * Search a recipe by name
+     * @param chicken
+     * @return Recipe
+     */
```

Comment:
Only add JavaDoc if it is providing value over and above what can be read from the method.